jabgui/src/main/java/org/jabref/gui/edit/CopyMoreAction.java

```diff
@@ -264,4 +274,70 @@ private void copyKeyAndLink() {
                     Long.toString(entries.size() - entriesWithKey.size()), Integer.toString(entries.size())));
         }
     }
+
+    private void copyField(StandardField field, String fieldDisplayName) {
+        List<BibEntry> selectedBibEntries = stateManager.getSelectedEntries();
+
+        List<String> fieldValues = selectedBibEntries.stream()
+                                                     .filter(bibEntry -> bibEntry.getField(field).isPresent())
+                                                     .map(bibEntry -> bibEntry.getField(field).orElse(""))
+                                                     .filter(value -> !value.isEmpty())
+                                                     .collect(Collectors.toList());
+
+        if (fieldValues.isEmpty()) {
+            dialogService.notify(Localization.lang("None of the selected entries have %0.", fieldDisplayName));
+            return;
+        }
+
+        final String copiedContent = String.join("\n", fieldValues);
+        clipBoardManager.setContent(copiedContent);
+
+        if (fieldValues.size() == selectedBibEntries.size()) {
+            // All entries had the field.
+            dialogService.notify(Localization.lang("Copied '%0' to clipboard.",
+                    JabRefDialogService.shortenDialogMessage(copiedContent)));
+        } else {
+            dialogService.notify(Localization.lang("Warning: %0 out of %1 entries have undefined %2.",
+                    Integer.toString(selectedBibEntries.size() - fieldValues.size()),
+                    Integer.toString(selectedBibEntries.size()),
+                    fieldDisplayName));
+        }
+    }
+
+    private void copyJournalField() {
+        List<BibEntry> selectedBibEntries = stateManager.getSelectedEntries();
+
+        List<String> journalValues = selectedBibEntries.stream()
+                                                       .filter(bibEntry ->
+                                                               bibEntry.getField(StandardField.JOURNAL).isPresent() ||
+                                                               bibEntry.getField(StandardField.JOURNALTITLE).isPresent())
```

Comment:
Use `getFieldOrAlias` instead of this hack. - I think, the method can be removed completely